### DISCUSSION TITLE
Live validation in livewire component

### DISCUSSION THREAD

**Discussion by sinnbeck** [filamentphp/filament#9050] [Help]
Has anyone gotten live validation working in a livewire component with a filament form?

I can see that it send a request to the backend but it does not call the `validate()` method it seems. I have tried setting a breakpoint in both livewire and filament `validate` methods, but neither is being hit.

Edit: It seems the same happens in a panel form. Does filament not support real time validation ?

**Comment by sinnbeck** [comment]
I can partially get it working by adding
```
    public function updated($field)
    {
        $this->validateOnly($field);
    }
```
But this does not work with file uploads as those gets a hash appended to the name

Edit: Got it working on the files as well, but I am unsure if this is how its meant to be done?
```
->afterStateUpdated(fn() => $this->validateOnly('data.files'))

**Comment by wq9578** [comment]
Live validation for both uploads and other fields: https://laraveldaily.com/post/filament-validate-one-form-field-live-before-submit
I haven't tested it for the upload case yet, but for other fields it works.
